skaha/src/main/java/org/opencadc/skaha/session/GetAction.java

```diff
@@ -95,19 +101,22 @@ public GetAction() {
     public void doAction() throws Exception {
         super.initRequest();
         if (requestType.equals(REQUEST_TYPE_SESSION)) {
+            String view = syncInput.getParameter("view");
             if (sessionID == null) {
-                // List the sessions
-                String typeFilter = syncInput.getParameter("type");
-                String statusFilter = syncInput.getParameter("status");
-                String view = syncInput.getParameter("view");
-                boolean allUsers = SESSION_LIST_VIEW_ALL.equals(view);
+                if (SESSION_VIEW_STATS.equals(view)) {
+                    reportStats();
```

Comment:
This is good, but the syncOutput operations can remain in doAction().  (the last two lines in reportStats())

Reply:
Refactored GetAction.